src/main/scala/bookbot/models/TitleAndAuthor.scala

```diff
@@ -0,0 +1,26 @@
+package bookbot.models
+
+import bookbot.models.errors.TransformError
+import bookbot.models.errors.Error
+import Error.ErrorOr
+
+case class TitleAndAuthorParseError(message: String) extends Error {
+  def getMessage = message
+}
+
+object TitleAndAuthor {
+  type TitleAndAuthor = (String, String)
+
+  def from(input: String): Either[Error, TitleAndAuthor] = {
+    val words = input.split("by", 2).map(_.trim).toList
+
+    words match {
+      case allStrings if noWordsIn(allStrings) => Left(TitleAndAuthorParseError("No search terms provided"))
+      case title :: author :: tail => Right(title, author)
+      case searchTerm :: tail => Right(searchTerm, "")
```

Comment:
This scenario gives me pause, as the return type alias of `TitleAndAuthor` is now a bit misleading since the `Title` is now a general search term and the `Author` is an empty string. As a user I do like that it will still search for something if I just give it a single word. I don't necessarily have a solution right now though 🤔